### PR TITLE
Feature/view mentor slots

### PR DESCRIPTION
**Pull request by PL9627**
Created an action to display slot availability, added cases for fetchMentorSlots reducer, started to fill in MentorSlotView component with fetchMentorSlots state, rerouted MentorSlotView from MiniMentorList to HeadmasterDashboard

### CODE REVIEW

**Comment by PL9627** [comment]
resolved the merge conflict but can't commit due to some error and warnings after pulling from main.

edit: fixed the err. MatchingCalendar was missing a bracket and the commit makes it look like I deleted and added a lot lines of code to fix it

**Review by jtwillo51 — DISMISSED** [review]

**Comment by jtwillo51** [comment]
Looks really good! I like how you make a commit for each change so I know exactly what to look for. Code looks clean and maintainable.

File: src/components/pages/Headmaster/HeadmasterDashboard.js [hunk 1]
```diff
@@ -129,6 +133,7 @@ const HeadmasterDashboard = props => {
                 path="/mentor-mentee-matching"
                 component={MatchingCalendar}
               />
+              <Route path="/avail" component={MentorSlotView} />
```
Comment (abdimohamud) [inline src/components/pages/Headmaster/HeadmasterDashboard.js:136]: Glad to see that you were able to get past that blocker with the routing!

**Review by tdavis1991 — DISMISSED** [review]
Everything looks good. It was quite easy to review from the comments you made.

**Review by abdimohamud — APPROVED** [review]

**Review by jtwillo51 — APPROVED** [review]

**Pull request merged**